assets/js/pages/declaration.js

```diff
@@ -0,0 +1,23 @@
+function checkPractitionerType() {
+  const practitionerType = document.getElementsByName('practitionerType');
+  const additional = document.getElementById('otherPractitionerType');
+
+  additional.style.display = 'none';
+
+  for (var i = 0, len = practitionerType.length; i < len; i++) {
+    if (practitionerType[i].checked) {
+      if (practitionerType[i].value === '4') {
```

Comment:
Can we use a short circuiting && instead of nested ifs, I know it's only two but I see nested if's as a bit more difficult to parse. 

I think the following would be more maintainable.

```javascript
if (practitionerType[i].checked && practitionerType[i].value === '4') 
```

Also it's how you do it in the `onclick` event right after so consistency is nice as well

Reply:
hmm... think I copied this from somewhere else, will make a note to go fix it there too.